x-pack/plugins/security_solution/server/usage/detections/detections_helpers.ts

```diff
@@ -4,12 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { SearchParams } from 'elasticsearch';
```

Comment:
Guessing this was an accidental import @rylnd?

Reply:
Its use was replaced by the custom `RuleSearchParams` below!